apple/RNGestureHandler.mm

```diff
@@ -495,6 +506,50 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
   return NO;
 }
 
+- (BOOL)areScrollViewRecognizersCompatible:(UIGestureRecognizer *)gestureRecognizer
+                           otherRecognizer:(UIGestureRecognizer *)otherGestureRecognizer
+{
+  if ([self isUIScrollViewPanGestureRecognizer:otherGestureRecognizer] &&
+      [gestureRecognizer isKindOfClass:[RNDummyGestureRecognizer class]]) {
+    UIScrollView *scrollView = [self retrieveScrollView:gestureRecognizer.view];
+    if (scrollView && scrollView == otherGestureRecognizer.view) {
+      return YES;
+    }
+  }
+
+  return NO;
+}
+
+// is UIPanGestureRecognizer and has scrollView property
+- (BOOL)isUIScrollViewPanGestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
+{
+  if ([gestureRecognizer isKindOfClass:[UIPanGestureRecognizer class]] &&
+      [gestureRecognizer respondsToSelector:@selector(scrollView)]) {
+    return YES;
+  }
+  return NO;
+}
+
+- (UIScrollView *)retrieveScrollView:(UIView *)view
```

Comment:
I believe this line breaks macOS compilation. You should be able to replace `UIView` with [`RNGHUIView`](https://github.com/software-mansion/react-native-gesture-handler/blob/5a671e9f0b02adfa4c565d0141cb6c01aaa3c0cc/apple/RNGHUIKit.h).

You will also need to do something about `UIScrollView` which will not exist on macOS. You should be able to use the `TARGET_OS_OSX` flag and conditionally replace it with `NSScrollView`.